utils/utils.py

```
@@ -21,6 +22,14 @@ def get_local_now() -> datetime:
     )
     return local_now
 
+def parse_json(data: Any, default: dict = {}) -> dict:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)